src/components/PostList.tsx

```diff
@@ -0,0 +1,51 @@
+import { Post } from '../types/Post';
+import { PostInfo } from './PostInfo';
+import { UserInfo } from './UserInfo';
+import { CommentList } from './CommentList';
+
+type Props = {
+  listOfPosts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ listOfPosts }) => (
+  <ol>
+    {listOfPosts.map((itemPost) => (
+      <li key={itemPost.id}>
+
+        <PostInfo
+          title={itemPost.title}
+          body={itemPost.body}
+        />
+
+        {itemPost.user && itemPost.user
```

Comment:
```suggestion
        {itemPost.user
```
Зачем два раза проверять?